interfaces/thirdparty/cp2k.py

```diff
@@ -499,20 +499,20 @@ def _parsemol(self):
         # make lines shorter
         inp = self.settings.input.force_eval.subsys
         # add cell information
-        nDim = len(self.molecule.lattice)
+        dim = len(self.molecule.lattice)
         keys = ['A', 'B', 'C']
         periodic = ['X', 'XY', 'XYZ']
-        for iDim in range(0, nDim):
-            inp.cell[keys[iDim]] = "{:} {:} {:}".format(
-                *self.molecule.lattice[iDim])
-        if nDim > 0:
-            inp.cell.periodic = periodic[nDim-1]
+        for i in range(dim):
+            inp.cell[keys[i]] = "{:.6f} {:.6f} {:.6f}".format(
```

Comment:
I'm not convinced that this is the correct place to resolve the issue.
Namely, a plain `"{:}"` should already format a float with an arbitrary number of decimals.

Reply:
Yup, suspicion confirmed: It's a QMFlows issue
https://github.com/SCM-NV/qmflows/blob/2dcc0879f0d18a4fc94b7bae90219ed5c7f3aef1/src/qmflows/packages/cp2k_package.py#L161-L162